src/solvers/dg_triangles/dg.jl

```diff
@@ -0,0 +1,287 @@
+# !!! warning "Experimental features"
```

Comment:
Could you please add this warning also to the example elixirs so that we can change things later without introducing breaking changes?

Reply:
Done. Also tagged the other source files with the same warning.